backend/controller/cronjobs/cronjobs_test.go

```diff
@@ -21,7 +21,6 @@ import (
 )
 
 func TestServiceWithMockDal(t *testing.T) {
-	t.Skip("TODO(matt2e): Fails -race, fix me")
```

Comment:
This one sometimes fails - you'll likely need to run it multiple times to reproduce.

Reply:
ahh, ok will retry more times